### PR TITLE
Added test script for local usage 

### PR DESCRIPTION
**Pull request by arjantijms**
This script works like the Jenkinsfile tests, but targets usage locally on the command line.

Update Readme to explain script usage

Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: gftest.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (smillidge) [inline gftest.sh:4]: Shouldn't this be the Eclipse Foundation and shouldn't the license be the EPL?

**Review by smillidge — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — APPROVED** [review]
Script seems to work (though the tests don't seem to like my local env - couldn't find some logs)

File: gftest.sh (hunk 1 above)
Comment (arjantijms) [reply gftest.sh:4]: Yes, it probably should. I'll update it, thx!

File: gftest.sh [hunk 2]
```diff
@@ -1,41 +1,17 @@
 #
-# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# Copyright (c) 2020 Eclipse Foundation and/or its affiliates. All rights reserved.
```
Comment (smillidge) [inline gftest.sh:2]: Strictly the copyright is yours if it is original work see https://www.eclipse.org/projects/handbook/#ip-copyright-headers you license the work to the Eclipse foundation under the EPL

**Review by smillidge — APPROVED** [review]

File: gftest.sh (hunk 2 above)
Comment (arjantijms) [reply gftest.sh:2]: Okay, I'll keep that in mind for a next commit. I copied the loop from another test file, though I was the one (and only one) to have written that loop.

**Pull request merged**